cmake/config/config_armv7l-tizen.cmake

```diff
@@ -0,0 +1,19 @@
+# Copyright 2017 Samsung Electronics Co., Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include(CMakeForceCompiler)
+
+set(CMAKE_SYSTEM_NAME Linux)
```

Comment:
At first, I thought Tizen is right for `CMAKE_SYSTEM_NAME` to distinguish between Linux and Tizen.

In addition, currently there are two kinds of Tizen: *Tizen Common* and  *Tizen RT*.
I think we may need to distinguish between _Tizen Common_ and *Tizen RT*.

Furthermore, I cannot find any occurrence that use `CMAKE_SYSTEM_NAME` or `CMAKE_SYSTEM_PROCESSOR` in libtuv. If it is true, I think we don't need another cmake file for tizen. We can use the general arm/linux config. (i.e.,  *config_arm-linux.cmake*)